docs/Installation.rst

```diff
@@ -1,178 +1,3 @@
- have to either overwrite the existing output file with
-
-    `mvs_tool -f`
-
-Or provide another output folder's path
-
-    `mvs_tool -o <path_to_other_output_folder>`
-
-Generate pdf report or an app in your browser to visualise the results of the simulation
-----------------------------------------------------------------------------------------
-
-To use the report feature you need to install extra dependencies first
-
-    `pip install multi-vector-simulator[report]`
-
-Generate a report after running a simulation
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Use the option `-pdf` in the command line  `mvs_tool` to generate a pdf report in the
- simulation's output folder (by default in `MVS_outputs/report/simulation_report.pdf`):
-
-    `mvs_tool -pdf`
-
-Generate only the figures of a simulation's results
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Use the option `-png` in the command line  `mvs_tool` to generate png figures of the results in the
- simulation's output folder (by default in `MVS_outputs/`):
-
-    `mvs_tool -png`
-
-
-post-processing
-^^^^^^^^^^^^^^^
-To generate a report of the simulation's results, run the following command **after** a simulation
- generated an output folder:
-
-    `mvs_report -i path_simulation_output_folder -o path_pdf_report`
-
-where `path_simulation_output_folder` should link to the folder of your simulation's output, or directly to a json file (default `MVS_outputs/json_input_processed.json`)
-and `path_pdf_report` is the path where the report should be saved as a pdf file.
-
-
-The report should appear in your browser (at http://127.0.0.1:8050) as an interactive Plotly Dash app.
-
-You can then print the report via your browser print functionality (ctrl+p), however the layout of
- the pdf report is only well optimized for chrome or chromium browser.
-
-It is also possible to automatically save the report as pdf by using the option `-pdf`
-
-    `mvs_report -i path_simulation_output_folder -pdf`
-
-By default, it will save the report in a `report` folder within your simulation's output folder default (`MVS_outputs/report/`).
-See `mvs_report.py -h` for more information about possible options. The css and images used to make the report pretty should be located under
-`report/assets`.
-
-Contributing and additional information for developers
-------------------------------------------------------
-
-If you want to contribute to this project, please read [CONTRIBUTING.md](https://github.com/rl-institut/multi-vector-simulator/blob/dev/CONTRIBUTING.md). For less experienced github users we propose a workflow. [Link](https://github.com/rl-institut/multi-vector-simulator/wiki/Examplary-Workflow).
-
-For advanced programmers: You can also use the dev version that includes the latest updates and changes, but which in turn might not be tested. You can find the CHANGELOG.md on this`page <https://github.com/rl-institut/multi-vector-simulator/blob/dev/CHANGELOG.md>`__.
```

Comment:
Alternatively, you dont put this into the readme.rst, but leave it as here. Readme would only need one additional point such as "To now execute the MVS with your personal files, execute".

Reply:
Can you explain a bit more? 
I should move the entire content from the section _Using the MVS_ onward to Installation.rst and then, add a line like you suggested above in README.rst, right?

Reply:
That was the idea... but we have to do major restructuring in the RTD anyway, so I would be fine with leaving it as is.